### PR TITLE
Admin Category Page Bug Fix

### PR DESCRIPTION
**Pull request by dezshino**
- Add cucumber and rspec unit tests for this bug
- Fix bug, categories can now be added or edited by admin

### CODE REVIEW

**Comment by kariabancroft** [comment]
the db/db_development and db/db_test files should not be in your commits

File: features/manage_categories.feature [hunk 1]
```diff
@@ -0,0 +1,33 @@
+Feature: Manage Categories
```
Comment (kariabancroft) [inline features/manage_categories.feature:1]: Nice job naming this feature for including many different scenarios

File: features/manage_categories.feature [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    Given I have no categories
+    When I follow "Categories"
+    And I fill in "Name" with "Foobar"
+    And I press "Save"
+    Then I should see "Category was successfully saved"
+    And I should see "Foobar"
+    And I should have 1 category
+
+  Scenario: Successfully edit categories
+    Given I have 1 category
```
Comment (kariabancroft) [inline features/manage_categories.feature:26]: Nice, i like this approach

**Comment by dezshino** [comment]
@kariabancroft The db/db_development and db/test files have been removed

**Comment by kariabancroft** [comment]
Looks good to close!

**Pull request closed without merging**